docs/xml-tools/walkthrough-using-xml-editor-features.md

```diff
@@ -52,7 +52,7 @@ The steps in this walkthrough show you how to create a new XML document. The wal
 
     - **employee** to add the root element.
 
-2. Select **<!--** to add a comment node and press **Enter**.
+2. Select **&lt;!--** to add a comment node and press **Enter**.
```

Comment:
Think this is a Markdownlint bug, that causes a false positive without the change, but it renders the same. I'll file an upstream issue